a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -247,12 +247,21 @@ def execute(self, vthunder, backup_vthunder, vrrp_status):
                 raise
 
             try:
-                bc = self.client_factory(backup_vthunder)
-                bc.system.action.set_vcs_device(2, 100)
-                bc.system.action.set_vcs_para("192.168.0.100", "255.255.255.0")
-                bc.system.action.vcs_enable()
-                bc.system.action.vcs_reload()
-                LOG.info("Configured the backup vThunder for aVCS")
+                attempts = 10
+                while attempts > 0:
+                    # TODO: Need this loop to be moved in acos_client with
```

Comment:
If you make a todo in code be sure to make an accompanying story or task in our backlog.

Reply:
I have raised a spike story on Jira [STACK-756](https://a10networks.atlassian.net/browse/STACK-756)